src/popup/router/pages/Send.vue

```diff
@@ -50,23 +50,15 @@
           <p class="primary-title primary-title-darker text-left my-5 f-16">
             {{ $t('pages.send.checkalert') }}
           </p>
-          <div class="info-group">
-            <label class="info-label">{{ $t('pages.send.sendingAddress') }}</label>
-            <span
-              data-cy="review-sendingAddress"
-              class="info-span"
-              @click="openTxExplorer(account.publicKey)"
-              >{{ account.publicKey }}</span
-            >
-          </div>
-          <div class="info-group">
-            <label class="info-label">{{ $t('pages.send.receivingAddress') }}</label>
-            <span
-              data-cy="review-receivingAddress"
-              class="info-span"
-              @click="openTxExplorer(form.address)"
-              >{{ form.address }}</span
-            >
+          <div
+            v-for="(value, key) in { sender: account.publicKey, recipient: form.address }"
+            :key="key"
+            class="info-group"
+          >
+            <label class="info-label">{{ wording[key] }}</label>
+            <span :data-cy="`review-${key}`" class="info-span" @click="openTxExplorer(value)">{{
+              value
+            }}</span>
           </div>
```

Comment:
This looks ok but a more component-oriented approach would be to extract InfoGroup component and exactly specify fields without a cycle:
```suggestion
          <InfoGroup
            :value="account.publicKey"
            :label="$t('pages.send.sender')"
          />
          <InfoGroup
            :value="form.address"
            :label="$t('pages.send.recipient')"
          />
```
the necessity to pass `data-cy` is just crap, e2e tests shouldn't require that

Reply:
Actually, we may apply `data-cy` on InfoGroup and to make selectors on e2e side a bit more complex.